### ISSUE TITLE
/option does not work on 3.2.6

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#18]
**Reported by HelLViS69 on 22 Aug 2007 14:26:02 UTC**
If you try to clean KVIrc history by using $option, it will not save the configuration<br/>

Using:<br/>
/echo $option(stringlistRecentChannels)<br/>
/option stringlistRecentChannels (or /option stringlistRecentChannels "")<br/>
does not affect main.kvc<br/><br/>
I've also tried with<br/>
/options.save<br/>
but nothing to do, main.kvc will not be saved and the variable still remains set

This bug was introduced with 3.2.6, as servers were splitted up in several networks

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/18

**Comment by DarthGandalf** [comment]
**Modified by anonymous on 22 Aug 2007 14:35:12**
Component: component1 → component2

**Comment by DarthGandalf** [comment]
**Modified by anonymous on 22 Aug 2007 14:35:59**
Owner: noone → Hellvis69

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 22 Aug 2007 15:08:31**
Owner: Hellvis69 → noone

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 12 Nov 2007 22:45:23**
Component: component2 → Core
Description:

``` diff
--- 
+++ 
@@ -1,10 +1,11 @@
-If you try to clean KVIrc history by using $option, it will not save the configuration
-Using:
-/echo $option(stringlistRecentChannels)
-/option stringlistRecentChannels (or /option stringlistRecentChannels "")
-does not affect main.kvc
-I've also tried with
-/options.save
+If you try to clean KVIrc history by using $option, it will not save the configuration[[BR]]
+
+Using:[[BR]]
+/echo $option(stringlistRecentChannels)[[BR]]
+/option stringlistRecentChannels (or /option stringlistRecentChannels "")[[BR]]
+does not affect main.kvc[[BR]][[BR]]
+I've also tried with[[BR]]
+/options.save[[BR]]
 but nothing to do, main.kvc will not be saved and the variable still remains set

 This bug was introduced with 3.2.6, as servers were splitted up in several networks

```

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 14 Nov 2007 20:12:47**
Summary: $option does not work on 3.2.6 → /option does not work on 3.2.6

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 12 Feb 2008 00:43:28**
Version: none → 3.2.6 Anomalies

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 20 Mar 2008 15:14:10**
Owner: noone → pragmaware

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 26 Mar 2008 17:47:00**
Version: 3.2.6 Anomalies → 3.4.0 Virgo

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 04 Jul 2008 15:27:47**
already fixed in 3.4

---

Resolution: none → fixed
Status: new → closed
